src/code.cloudfoundry.org/cf-tcp-router/config/config_test.go

```diff
@@ -323,6 +323,41 @@ var _ = Describe("Config", Serial, func() {
 			})
 		})
 
+		Context("with invalid cert and key and enableCertCreation set to False", func() {
+			BeforeEach(func() {
+				cfg, err = config.New("fixtures/valid_frontend_cert.yml", false)
+			})
+
+			It("loads config without error", func() {
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("adds the certs and keys to the expected directories", func() {
+				Expect(err).NotTo(HaveOccurred())
+				Expect(cfg.FrontendTLS).To(HaveLen(2))
+
+				Expect(cfg.FrontendTLS[0]).To(Equal(config.FrontendTLSConfig{
+					Enabled:        true,
+					CertificateDir: filepath.Join(tmpDir, "prod"),
+				}))
+
+				Expect(cfg.FrontendTLS[1]).To(Equal(config.FrontendTLSConfig{
+					Enabled:        true,
+					CertificateDir: filepath.Join(tmpDir, "dev"),
+				}))
+			})
```

Comment:
Why are certs being added when enableCertCreation is False?

Reply:
Please see comment:
https://github.com/cloudfoundry/routing-release/pull/508/files#r2316441969